<!DOCTYPE html>
	<head>
    <meta charset = "utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link href="https://fonts.googleapis.com/css?family=Lato:300,400" rel="stylesheet"> 
      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" crossorigin="anonymous">
  <link href = "../index.css" rel = "stylesheet">
  <!-- Bootstrap CSS -->
  <link href="notebook2.css" rel="stylesheet">
  <!-- Custom CSS -->
  <link href="notebook.css" rel="stylesheet">
  <!--script src="notebook.js"></script-->
   
  
    </head>

	<body>
        <div class="page-wrapper">
          <div class="nav-wrapper">
            <div class="grad-bar"></div>
            <nav class="navbar">
              <img src="../igem iisertvm logo (1).png" alt="iGEM Logo" height="200">
              <div class="menu-toggle" id="mobile-menu">
                <span class="bar"></span>
                <span class="bar"></span>
                <span class="bar"></span>
              </div>
              <ul class="nav no-search">
                <li class="nav-item"><a href="index.html">Home</a></li>
                <li class="nav-item">
                  <div class="dropdown">
                    <button class="dropbtn">Team
                      <i class="fa fa-caret-down"></i>
                    </button>
                    <div class="dropdown-content">
                      <a href="Team.html">Team</a>
                      <a href="Attribution.html">Attributions</a>
                      <a href="Collaboration.html">Collaborations</a>
                </li>
                <li class="nav-item">
                  <div class="dropdown">
                    <button class="dropbtn">Project
                      <i class="fa fa-caret-down"></i>
                    </button>
                    <div class="dropdown-content">
                      <a href="Project/comm.html">Communication</a>
                      <a href="Project/contribution.html">Contribution</a>
                      <a href="Project/description.html">Description</a>
                      <a href="Project/engineering.html">Engineering</a>
                      <a href="Project/experiment.html">Experiments</a>
                      <a href="Project/implementation.html">Implementation</a>
                      <a href="Project/notebook.html">Notebook</a>
                      <a href="Project/partnership.html">Partnership</a>
                      <a href="Project/proof.html">Proof of Concept</a>
                      <a href="Project/results.html">Results</a>
      
      
                    </div>
                  </div>
                </li>
                <li class="nav-item"><a href="Parts.html">
                    <div class="dropdown">
                      <button class="dropbtn">Parts
                        <i class="fa fa-caret-down"></i>
                      </button>
                      <div class="dropdown-content">
      
                        <a href="Parts/partcollection.html">Parts Collection</a>
                        <a href="Parts.html">Parts</a>
                        <a href="Parts/Improve.html">Improve</a>
                  </a></li>
                <li class="nav-item"><a href="Results.html">Safety</a></li>
                <li class="nav-item"><a href="iHP.html">
                  <div class="dropdown">
                    <button class="dropbtn">Human Practices
                      <i class="fa fa-caret-down"></i>
                    </button>
                    <div class="dropdown-content">
      
                      <a href="iHP.html">Human Practices</a>
                      <a href="iHP1.html">Human Practices Timeline</a>
                    </a></li>
                <li class="nav-item"><a href="Results.html"></a></li>
      
      
              </ul>
            </nav>
          </div>
            </div>
            <div class="layout">
              <div class="outline blueblock">
                  <div></div>
                  <div class="container container-custom">
            
                  <h1>Biolab</h1><br>
            
                  <div class="row gy-5 gx-0 pb-5 justify-content-end">
                      <div class="col-1 d-flex justify-content-center order-2">
                          <div class="bar" style="height: 100%;"></div> 
                      </div>
                        <div class="col-10 order-1">
                          <div class="row gx-5">
                            <div class="col-12 mb-3">
                              <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 21</u></h5>
                <p class="padding_notebook" style="margin:left"><i>Weekly Goal: </i>Ordering our first gBlocks we will use for cloning and reviving glycerol stocks we have received to acquire plasmids we will use.</p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                    <b>Monday</b>
                <ul>
                  <li> Ordered the gBlock coding for IL-10, as well as PCR primers for this gBlock via IDT (<a class="link" href=" https://2022.igem.wiki/tu-eindhoven/primers_gblocks">Primer list</a>). The IL-10 was obtained via Uniprot (<a class="link" target="_blank" href="https://www.uniprot.org/uniprotkb/P22301/entry">UniProt ID: P22301</a>)</li>
                  <li> Start with reviving glycerol stocks (pLS13 (<a href="http://parts.igem.org/Part:BBa_K4160016" class="link" target="_blank">BBa_K4160016</a>), pLS15 (<a href="http://parts.igem.org/Part:BBa_K4160005" class="link" target="_blank">BBa_K4160005</a>), pLeo619 <a href="http://parts.igem.org/Part:BBa_K4160008" class="link" target="_blank">BBa_K4160008</a>), 4_P3, 4_P4) we have received which were stored in the -80 ॰C freezer.</li>
                </ul>
                
                <b>Tuesday</b>
                <ul>
                  <li> Colony picking of the colonies that have grown on the agar plates (Figure 1). These colonies will be used for plasmid preparation by starting small cultures today. As seen in figure 1, the plate for 4_P4 shows 2 different types of colonies growing. Continued with small cultures with both types of colonies.
                    <br>
                    <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w21/tu.jpg"></center>
                    <br>
                    <figcaption style="font-size:13px"><b>Figure 1 | LB-Agar plates.</b> NovaBlue cells transformed with either pLeo619, pLS13, pLS15, 4-P3, and 4-P4 plasmid. </figcaption>
                    <br>
                    </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday and subsequent concentration determination by Nanodrop spectrophotometer.</li>			
                </ul>
                </div>
            </div>
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 23</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Start with the first cloning (pLS13-IL-10). Prepare antibiotics and bacterial growth medium.                                        </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                    <b>Tuesday</b>
                <ul>
                  <li> Dissolving gBlock and primers to final concentrations. Final concentration for the gBlock is 25 ng/µL. Final molarity of primers are 100 µM (main stock) and 10 µM (working concentration).</li>
                  <li> gBlock amplification via PCR, followed by PCR cleanup and subsequent concentration determination via Nanodrop Spectrophotometer. As an example, data received from Nanodrop Spectrophotometer is seen in Table 1.
                  <br><br>
                  <figcaption style="font-size:13px"><b> Table 1 | Concentration determination IL-10 gBlock </b> The IL-10 gBlock was PCR amplified, and subsequently the concentration was determined based on 260 nm absorption values through a Nanodrop photospectrometer. </figcaption>
                  <br>
                  <center><img class="image-tofull" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w23/t1.png"></center>
                  <br>
                  </li>
                  <li> Restriction digest on plasmid (pLS13) and insert (IL-10, (<a href="http://parts.igem.org/Part:BBa_K4160007" class="link" target="_blank">BBa_K4160007</a>)). For digestion, XbaI (NEB) and Klf1 (ThermoFisher) are used. Restriction products were stored in fridge (4 ॰C). </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Re-did restriction digestion from yesterday because some errors were made (the rSAP enzyme, a phosphatase, was added to both the plasmid as insert, making a ligation impossible due to lack of 5’ and 3’ phosphates. Moreover, remeasured plasmid concentrations on the Nanodrop spectrophotometer). Again XbaI (NEB) and Klf1 (ThermoFisher) are used for digestion. The rSAP phosphatase was only added to the plasmid. </li>
                  <li> DNA gel electrophores was used to separate the different restriction products (Figure 2). Gel was stored in the fridge overnight (4 ॰C).
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w23/we.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b>Figure 2 | Agarose gel for DNA separation of digested pLS13 and IL-10.</b> For digestion of the plasmid and insert, XbaI (NEB) and KlfI (ThermoFisher) were used. The outlined DNA bands were cut out and purified . Plasmid and insert DNA length after digestion are 3466 bp and 544 bp, respectively. Agarose gel (1%), ran for 1 hour at 100 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Gel extraction (Qiagen kit) was used to purify the digested DNA from the gel. Subsequently, the concentrations were determined using the Nanodrop Spectrophotometer.</li>
                  <li> Ligation of digested products using T4 DNA Ligase (NEB). Ligated product was stored in fridge overnight (4 ॰C).</li>
                  <li> Ampicillin  stock (1000x) was made.</li>
                  <li> LB-Agar plates containing ampicillin were made. </li>
                  <li> LB growth medium including ampicillin was made.</li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Transformation of ligated product in TOP10 Competent Cells. Moreover, a negative control was added (using only the digested plasmid, no insert), to check for false positives. Transformants were plated on (prewarmed) LB-Agar plates and grew over the weekend at room temperature. </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 24</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Determine if the transformation of the plasmid pLS13-IL-10 was successful. Order and prepare new primers to sequence verify the ligated plasmid. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                    <b>Tuesday</b>
                <ul>
                  <li> Performed colony PCR to determine which colony on the LB-agar plate had an insert (Figure 3), as colonies are also seen on the negative control. This ensures that no false positives are used later on. 
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w24/tu1.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b>Figure 3 | LB-Agar plates.</b> Top10 cells transformed with either pLS13, or pLS13-IL-10 plasmid. </figcaption>
                  <br>
                  </li>
                  <li> After a PCR reaction with primers specific for IL-10, 10 samples were put on an agarose gel to read out (Figure 4). 
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w24/tu2.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 4 | Agarose gel analysis on colony PCR samples transformed with pLS13-IL-10 plasmid. </b> The gel shows 10 colony samples after a PCR reaction in which primers, specific for IL-10, are used. Clear bands are seen in each colony sample at the expected Kb (534 bp). Due to a possible contamination of IL-10 DNA, in the digested plasmid control, a band is also seen at 534 bp. Agarose gel (1%), ran for 45 hour at 120 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                  <li> Small cultures were started by using colony PCR samples 1 to 5. Small cultures grew overnight at 37 ॰C, 250 rpm. </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Took the small cultures out of the incubator and stored in the fridge (4 ॰C).</li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Ordered sequencing primers for plasmids and another gBlock (<a class="link" href=" https://2022.igem.wiki/tu-eindhoven/primers_gblocks">Primer list</a>). The primers are specificically designed for the plasmids (pLS13 (<a href="http://parts.igem.org/Part:BBa_K4160016" class="link" target="_blank">BBa_K4160016</a>), pLS15 (<a href="http://parts.igem.org/Part:BBa_K4160005" class="link" target="_blank">BBa_K4160005</a>), pLeo619 (<a href="http://parts.igem.org/Part:BBa_K4160008" class="link" target="_blank">BBa_K4160008</a>), 4_P3, 4_P4. The gBlock sequence codes for the protein PR3. The PR3 sequence was obtained from Uniprot (<a class="link" target="blank" href="https://www.uniprot.org/uniprotkb/P24158/entry">UniProt ID: P24158</a>).</li>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday and subsequent concentration determination by Nanodrop spectrophotometer. Plasmids are stored in the freezer (-20 ॰C)</li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Prepared IL-10 PCR primer working condition stock (10 µM) from the main stock (100 µM). </li>
                  <li> Made sequencing samples of our cloned plasmids from yesterday that will be sent to BaseClear for sequencing.</li>
                  <li> Transformation of multiple plasmids (pLS13 (<a href="http://parts.igem.org/Part:BBa_K4160016" class="link" target="_blank">BBa_K4160016</a>), pLS15 (<a href="http://parts.igem.org/Part:BBa_K4160005" class="link" target="_blank">BBa_K4160005</a>), pLeo619 (<a href="http://parts.igem.org/Part:BBa_K4160008" class="link" target="_blank">BBa_K4160008</a>), 4_P3 ) in Subcloning Efficiency™ DH5α™ Chemically Competent Cells. Transformants were plated on (prewarmed) LB-Agar plates and grew over the weekend at room temperature. </li>
                </ul>
                </div>
            </div>
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 25</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Multiply plasmids needed for the cell experiments. Prepare small cultures and glycerol stocks of pLS13-IL-10. Preparation of LB-Agar plates. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Reviving glycerol stocks (0_P3, 4_P3_varG, 25_P3_va rG) we have received which were stored in the -80 ॰C freezer </li>
                  <li> Started small cultures from the colonies on the LB-agar plates from all but one of the transformations done on 17-06-22 (Figure 5). Did not continue with 4_P3 because this was not the 4_P3_varG variant we found today.
                  <br><br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w25/mo.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 5 | LB-Agar plates. </b>DH5α subcloning efficiency cells transformed with either pLeo619, pLS13 (SEAP), pLS15 (STAT3), or 4-p3 plasmid. </figcaption>
                  <br>
                  </li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Prepared the primers we ordered on 16-06. Final molarity of primers are 100 µM (main stock) and 10 µM (working concentration).  </li>
                  <li> Started small cultures from the colonies on the LB-agar plates from the glycerol stock revival of 20-06-22 (Figure 6).
                  <br><br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w25/tu.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b>Figure 6 | LB-Agar plates. </b> NovaBlue, XL1-Blue and XL10-Gold cells transformed with either 0_P3, 4_P3, or 25_P3 plasmid.</figcaption>
                  <br>
                  </li>
                  <li> Made glycerol stocks from the small cultures (pLS13, (<a href="http://parts.igem.org/Part:BBa_K4160016" class="link" target="_blank">BBa_K4160016</a>), pLS15 (<a href="http://parts.igem.org/Part:BBa_K4160005" class="link" target="_blank">BBa_K4160005</a>), pLeo619 (<a href="http://parts.igem.org/Part:BBa_K4160008" class="link" target="_blank">BBa_K4160008</a>)) we started yesterday (20-06). After snap freezing them, put them in the freezer (-80 ॰C). </li>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday (20-06) and subsequent concentration determination by Nanodrop spectrophotometer.  </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Made glycerol stocks from the small cultures (0_P3, 4_P3vG, 25_P3vG ) we started yesterday (21-06). After snap freezing them, put them in the freezer (-80 ॰C). </li>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday (21-06) and subsequent concentration determination by Nanodrop spectrophotometer. </li>
                  <li> LB-Agar plates containing ampicillin were made. </li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Due to low plasmid concentrations measured last Wednesday, another transformation was done in Subcloning Efficiency™ DH5α™ Chemically Competent Cells (0_P3, 4_P3vG, 25_P3vG, 4_P4). This is expected to increase the plasmid concentrations. Transformants were plated on (prewarmed) LB-Agar plates and grew over the weekend at room temperature. </li>
                  <li> Reviving glycerol stocks (4_P 4) we have received which were stored in the -80 ॰C freezer.</li>
                </ul>
                </div>
            </div>
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 26</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Start with creating the library of plasmids for the second cloning. Start with digestion of plasmids and insert.  </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Started small cultures from the colonies on the LB-agar plates from the transformation of 24-06-22 (Figure 7)
                  <br><br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w26/mo1.jpg"></center>
                  <br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w26/mo2.jpg"></center>
                  <br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w26/mo3.jpg"></center>
                  <br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w26/mo4.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b>Figure 7 | LB-Agar plates. </b> DH5α and NovaBlue cells transformed with either 4_P4, 0_P3, 4_P3, or 25_P3 plasmid. </figcaption>
                  <br>
                  </li>
                  <li> Transformation was performed in Subcloning Efficiency™ DH5α™ Chemically Competent Cells (2_P3). Transformants were plated on (prewarmed) LB-Agar plates and grew overnight at 37 ॰C. </li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Made glycerol stocks from the small cultures (0_P3, 4_P3vG, 25_P3vG, 4_P4 ) we started yesterday (20-06). After snap freezing them, put them in the freezer (-80 ॰C). </li>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday (27-06) and subsequent concentration determination by Nanodrop spectrophotometer. As seen in Figure 8, small cultures containing the 4_P4 plasmid did not grow. 
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w26/tu.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b>Figure 8 | Small cultures of the 4_P4 transformed cultures. </b> NovaBlue cells, transformed with 4_P4 plasmids. Small cultures were unable to grow.  </figcaption>
                  <br>
                  </li>
                  <li> Started small cultures from the colonies on the LB-agar plates from the transformation of 28-06-22 (2_P3).</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Dissolving PR3 gBlock we ordered on 16-06 to final concentrations. Final concentration for the gBlock is 25 ng/µL. </li>
                  <li> PR3 gBlock amplification via PCR, followed by PCR cleanup and subsequent concentration determination via Nanodrop Spectrophotometer. Performed the PCR. </li>
                  <li> Made glycerol stocks fr om the small cultures (2_P3) we started yesterday (28-06). After snap freezing them, put them in the freezer (-80 ॰C). </li>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday (28-06) and subsequent concentration determination by Nanodrop spectrophotometer. </li>
                  <li> Transformation was performed in Subcloning Efficiency™ DH5α™ Chemically Competent Cells (4_P4, different aliquot). Transformants were plated on (prewarmed) LB-Agar plates and grew overnight at 37 ॰C. </li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Started small cultures from the colonies on the LB-agar plates from the transformation of 29-06-22 (4_P4).</li>
                  <li> Restriction digest on plasmid (2_P3 and 4_P4vG) and insert (PR3, <a href="http://parts.igem.org/Part:BBa_K4160004" class="link" target="_blank">BBa_K4160004</a>). For digestion, BamHI (NEB) and EcoRI (NEB) are used. rSAP, a phosphatase, was only added to the plasmid for 5’ and 3’ dephosphorylation.</li>
                  <li> DNA gel electrophoresis was used to separate the different restriction products. (Figure 9). The 2-log NEB ladder was added as reference.</li>
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w26/th1.jpg"></center>
                  <br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w26/th2.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b>Figure 9 | Agarose gels for DNA separation of digested 0_P3, 2_P3, 4_P3, 25_P3, and PR3. </b> For digestion of the plasmid and insert BamHI (NEB) and EcoRI-HF (NEB) were used. The outlined DNA bands were cut out and purified. Plasmid DNA length after digestion for -0_P3, 2_P3, 4_P3 and 25_P3 are all round 7000 bp. Digested PR3 is 663 bp. Agarose gel (1%), ran for 1 hour at 100 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                  <li> Gel extraction (Qiagen kit) was used to purify the digested DNA from the gel. Subsequently, the concentrations were determined using the Nanodrop Spectrophotometer.  </li>
                </ul> 
                
                <b>Friday</b>
                <ul>
                  <li> Made glycerol stocks from the small cultures (4_P4) we started yesterday (30-06). After snap freezing them, put them in the freezer (-80 ॰C).</li>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday (30-06) and subsequent concentration determination by Nanodrop spectrophotometer.  </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 27</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Ligate the digested products and subsequent transformation. Verification of ligated products via colony PCR. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                    <b>Tuesday</b>
                <ul>
                  <li> Ligation of digested products of 30-06 using T4 DNA Ligase (NEB).</li>
                  <li> Transformation of these ligated products in TOP10 Competent Cells. Moreover, a negative control was added (using only the digested plasmid, no insert), to check for false positives. Transformants were plated on (prewarmed) LB-Agar plates and grew overnight at 37 ॰C. </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Performed colony PCR  on 0_PR3 (<a href="http://parts.igem.org/Part:BBa_K4160009" class="link" target="_blank">BBa_K4160009</a>, plate 1) and 31_PR3 (<a href="http://parts.igem.org/Part:BBa_K4160011" class="link" target="_blank">BBa_K4160011</a>, plate 4) to determine which colony on the LB-agar plate had an insert (Figure 10), as colonies are also seen on the negative control. This ensures that no false positives are used later on. No colonies grew on 4_PR3 (2) and 8_PR3 (3).
                  <br><br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w27/we.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 10 | LB-Agar plates. </b> Top10 cells transformed with either 0_PR3, 4_PR3, 8_PR3, or 31_PR3 plasmid. </figcaption>
                  </li>
                  <li> After a PCR reaction with primers specific for PR3, 5 samples per plate were put on an agarose gel to read out. (Figure 11).
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w27/we2.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 11 |  Agarose gel analysis on colony PCR samples transformed with 0_PR3, and 31_PR3 plasmid.  </b> The gel shows two times seven colony samples after a PCR reaction. In this reaction primers, specific for PR3, are used. Clear bands are seen in each colony sample at the expected Kb (663 bp). All controls show no bands due to the PR3 not being present. Agarose gel (1%), ran for 45 hour at 120 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator. </figcaption>
                  <br>
                  </li>
                  <li> Started small cultures from the colonies on the LB-agar plates from the transformation of 05-07-22 (0_PR3 and 31_PR3).</li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Made glycerol stocks from the small cultures (0_PR3 and 31_PR3) we started yesterday (06-07). After snap freezing them, put them in the freezer (-80 ॰C). </li>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday (06-07) and subsequent concentration determination by Nanodrop spectrophotometer.  </li>
                </ul>
                </div>
            </div>
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 28</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Repeat of digestion, ligation, and transformation, using last weeks plasmids. Sequence verification of starting plasmids and ligation products. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> As the SEAP plasmid stock almost ran out, we started small cultures from the glycerol stock of 21-06-22 (SEAP 2).</li>
                  <li> As we did not have any glycerol stocks of the first cloning (pLS-13 + IL-10) we started a transformation in Subcloning Efficiency™ DH5α™ Chemically Competent Cells (pLS-13 + IL-10 from 16-06-22, ). Transformants were plated on (prewarmed) LB-Agar plates and grew overnight at 37 ॰C.</li>
                  <li> Re-done restriction digest on plasmid (2_P3 and 4_P4vG) and insert (PR3), because no colonies grew on 06-07. For digestion, BamHI (NEB) and EcoRI (NEB) are used. rSAP, a phosphatase, was only added to the plasmids for 5’ and 3’ dephosphorylation. </li>
                  <li> DNA gel electrophoresis was used to separate the different restriction products. (Figure 12). The 2-log NEB ladder was added as reference.
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w28/mo.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 12 | Agarose gels for DNA separation of digested 2_P3, 4_P3, and PR3. </b> For digestion of the plasmid and insert BamHI (NEB) and EcoRI-HF (NEB) were used. After digestion, clear bands were seen for PR3 and 2_P3 and 4_P3 at 663 bp and 7000 bp, respectively. The outlined DNA bands were cut out and purified. Agarose gel (1%), ran for 1 hour at 100 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                  <li> Gel extraction (Qiagen kit) was used to purify the digested DNA from the gel. Subsequently, the concentrations were determined using the Nanodrop Spectrophotometer. </li>
                  <li> Ligation of digested products of today using T4 DNA Ligase (NEB). Incubated reaction overnight at 4 ॰C. This ligates the parts to 2_PR3 and 8_PR3.</li>
                  <li> PR3 gBlock amplification via PCR, followed by PCR cleanup and subsequent concentration determination via Nanodrop Spectrophotometer. </li>
                  <li> Made sequencing samples of our cloned plasmids from 07-07 (0_PR3 and 31_PR3) that will be sent to BaseClear for sequencing.</li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Transformation of ligated product from 11-07 in Subcloning Efficiency™ DH5α™ Chemically Competent Cells (2_PR3 and 8_PR3). Moreover, a negative control was added (using only the digested plasmid, no insert), to check for false positives. Transformants were plated on (prewarmed) LB-Agar plates and grew grew overnight at 37 ॰C.</li>
                  <li> Started small cultures from the colonies on the LB-agar plates from the transformation of 11-07 (pLS-13 + IL-10).</li>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday (11-07) and subsequent concentration determination by Nanodrop spectrophotometer. </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Performed colony PCR  on 2_PR3 (1) and 8_PR3 (4) to determine which colony on the LB-agar plate had an insert (Figure 13). This ensures that no false positives are used later on.
                  <br><br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w28/we.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 13 | LB-Agar plates.</b> DH5α cells transformed with either 2_P3, or 4_PR3 plasmid. </figcaption>
                  <br>
                  </li>
                  <li> After a PCR reaction with primers specific for PR3, 5 samples per plate were put on an agarose gel to read out. (Figure 14). As seen in the gel, unexpected results are seen. This was due to adding the wrong primer.
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w28/we2.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b>Figure 14 | Agarose gel analysis on colony PCR samples transformed with 2_PR3, and 8_PR3 plasmid. </b>  The gel shows four (2_PR3) and eight (8_PR3) colony samples after a PCR reaction. A mistake was made, which led us to add the wrong primer. This explains the bands seen at the wrong base pair length (expected bands at 663 bp). Agarose gel (1%), ran for 45 hour at 120 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                  <li> Made glycerol stocks from the small cultures (pLS13 + IL-10) we started yesterday (12-07). After snap freezing them, put them in the freezer (-80 ॰C). </li>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday (12-07) and subsequent concentration determination by Nanodrop spectrophotometer.  </li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Made glycerol stocks from the small cultures (2_PR3 and 4_PR3) we started yesterday (13-07). After snap freezing them, put them in the freezer (-80 ॰C).</li>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday (13-07) and subsequent concentration determination by Nanodrop spectrophotometer.  </li>
                  <li> Made sequencing samples of our cloned plasmids from today (2_PR3 and 8_PR3) that will be sent to BaseClear for sequencing.</li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 29</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Analysis of sequencing results from the starting plasmids. Design new cloning strategies followed by repeating digestion, ligation, and transformation. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> After receiving Sanger sequencing results from Baseclear we changed some nomenclatures:
                  <ul>
                    <li> 0_PR3 had mutations, so decided to remove this plasmid from our use.</li>
                    <li> The expected 2_PR3 showed to have a linker inside the part that was cut out after digestion. After the ligation, no linker is present anymore, so from now on we use the name 0_PR3. </li>
                    <li> 8_PR3 was correct, but to prevent confusion we will call it 8_PR3.</li>
                    <li> 31_PR3 was incorrect due to an unexpected linker sequence, also including multiple restriction sites. This forced us to change our restriction digestion design. </li>
                  </ul> 
                  </li>
                  <li> A new cloning strategy is designed to acquire 31_PR3 (Figure 15).
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/engi-suc/9.png"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 15 | New cloning strategy (3-part ligation). </b> To acquire the 31_PR3 plasmid, we have designed a new cloning strategy. By cutting the 25_P3 plasmid with different combinations of restriction enzymes, a backbone part (25_P3 BB, left plasmid) and receptor and linker part (EpoR + Linker, right plasmid) are restricted. After the restriction of the PR3 gBlock with the correct enzymes, a 3-part ligation can be executed to acquire the correct plasmid.</figcaption>
                  <br>			
                  </li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Restriction digest on plasmid (25_P3) and insert (PR3). For digestion, XbaI (NEB), NotI (NEB) and EcoRI (NEB) are used. </li>
                  <li> DNA gel electrophoresis was used to separate the different restriction products (Figure 16). The 2-log NEB ladder was added as reference.
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w29/tu.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 16 | Agarose gels for DNA separation of digested 25_P3. </b> For digestion of 25_P3 BB, NotI (NEB) and BamHI were used. For digestion of EpoR + linker, NotI (NEB) and EcoRI (NEB) were used. Clear, expected bands were seen at 5600 bp (25_P3 BB) and around 1687 bp (EpoR + Linker). The outlined DNA bands were cut out and purified. Agarose gel (1%), ran for 1 hour at 100 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                  <li> Gel extraction (Qiagen kit) was used to purify the digested DNA from the gel. Moreover, PCR cleanup (Qiagen kit) was used for filtering out digested PR3. Subsequently, the concentrations were determined using the Nanodrop Spectrophotometer. </li>
                  <li> Ligation of digested products of today using T4 DNA Ligase (NEB). Incubated reaction overnight at 4 ॰C. This ligates the parts to 31_PR3.</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Transformation of the ligated products from 19-07 in TOP10 Competent Cells. Moreover, a negative control was added (using only the digested plasmid, no insert), to check for false positives. Transformants were plated on (prewarmed) LB-Agar plates and grew overnight at 37 ॰C. </li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Ligation was unsuccessful because no colonies grew on the LB-Agar plates we started yesterday (20-07). Therefore, redid the ligation with the digested products of 19-07. Ligation of digested products of today using T4 DNA Ligase (NEB). Incubated reaction overnight at 4 ॰C. This ligates the parts to 31_PR3.</li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Transformation of the ligated products from 21-07 in TOP10 Competent Cells. Moreover, a negative control was added (using only the digested plasmid, no insert), to check for false positives. Also, 8_PR3 was transformed as a postive control for the transformation. Transformants were plated on (prewarmed) LB-Agar plates and grew over the weekend at room temperature. </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 30</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Repeat the digestion, ligation, and transformation for the longest linker.                                               </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Ligation and/or transformation was unsuccessful because no colonies grew on the LB-Agar plates we started yesterday (20-07) (Figure 17). Therefore, we redid the whole experiment, starting with the digestion of 25_P3 and newly made PR3. 
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w30/mo.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 17 | LB-Agar plates. TOP10 cells transformed with either 8_PR3, or 31_PR3 plasmid.  </b>TOP10 cells transformed with either 8_PR3, or 31_PR3 plasmid. </figcaption>
                  <br>
                  </li>
                  <li> PR3 gBlock amplification via PCR, followed by PCR cleanup and subsequent concentration determination via Nanodrop Spectrophotometer. PCR results showed to be really low. </li>
                  <li> Redo of PR3 gBlock amplification via PCR (using newly diluted primers), followed by PCR cleanup and subsequent concentration determination via Nanodrop Spectrophotometer.</li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Restriction digest on plasmid (25_P3) and insert (PR3). For digestion, XbaI (NEB), NotI (NEB) and EcoRI (NEB) are used. </li>
                  <li> DNA gel electrophoresis was used to separate the different restriction products (Figure 18). The 2-log NEB ladder was added as reference. 
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w30/tu.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 18 | Agarose gels for DNA separation of digested 25_P3, and PR3. </b>For digestion of 25_P3 BB, NotI (NEB) and BamHI (NEB) were used. For digestion of EpoR + linker, NotI (NEB) and EcoRI (NEB) were used. For digestion of PR3 BamHI (NEB) and EcoRI (NEB) were used. Clear, expected bands were seen at 5600 bp (25_P3 BB), around 1687 bp (EpoR + Linker) and 663 bp (PR3). The outlined DNA bands were cut out and purified. Agarose gel (1%), ran for 1 hour at 100 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                  <li> Gel extraction (Qiagen kit) was used to purify the digested DNA from the gel. Subsequently, the concentrations were determined using the Nanodrop Spectrophotometer. </li>
                  <li> Ligation of digested products of today using T4 DNA Ligase (NEB). Incubated reaction at 16 ॰C for 1.5 hours. This ligates the parts to 31_PR3.</li>
                  <li> Transformation of the ligated products of today  in TOP10 Competent Cells. Moreover, a negative control was added (using only the digested plasmid, no insert), to check for false positives. Also, pLeo619 from 25-05 was transformed as a positive control for the transformation. Transformants were plated on (prewarmed) LB-Agar plates and grew overnight at 37 ॰C.</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Ligation did not work as we saw nothing growing on the 31_PR3 plate. Transformation was successful as the positive control showed colonies that grew (Figure 19).
                  <br><br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w30/we1.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 19 | LB-Agar plates. </b> TOP10 cells transformed with either pLeo619, or 31_PR3 plasmid.</figcaption>
                  <br>
                  </li>
                  <li> Designed a new cloning strategy to acquire 31_PR3, this time via Gibson Assembly (Figure 20). 
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/engi-suc/12.png"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 20 | New cloning strategy (Gibson Assembly). </b>  As the 3-part ligation cloning strategy did not work, we wanted to try Gibson Assembly. We designed multiple primers and came up with the strategy which is seen in the figure. </figcaption>
                  <br>
                  </li>
                  <li> Ordered primers specifically designed for Gibson Assembly (<a class="link" href=" https://2022.igem.wiki/tu-eindhoven/primers_gblocks">Primer list</a>).</li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 31</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Try a different cloning strategy: Gibson Assembly to clone the 31_PR3 plasmid.                              </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Prepared Gibson assembly primers to working condition stock (10 µM) from the main stock (100 µM)</li>
                  <li> Started the PCR reaction for plasmid linearization (25_P3) and overlap extension (on PR3 gBlock). This should result in the parts we need for Gibson assembly.</li>
                  <li> Subsequently started PCR cleanup, followed by concentration determination via Nanodrop Spectrophotometer. </li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Started the Gibson assembly with samples from yesterday. </li>
                  <li> Transformation of Gibson Assembly product in TOP10 Competent Cells. Transformants were plated on (prewarmed) LB-Agar plates and grew overnight at 37॰ C</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> No transformants grew overnight, making us think that the Gibson Assembly did not work (Figure 21). We kept our PCR product from 01-08, so next we wanted to analyze if the PCR reaction did work.
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w31/we.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 21 | LB-Agar plates. </b> TOP10 cells transformed with 31_PR3 plasmid. </figcaption>
                  <br>
                  </li>
                  <li> To check if the PCR was correct, we put the PCR product on an Agarose gel. We expect clear bands at 0.7 kb (PR3) and 6.7 kb (plasmid) (Figure 28). Gel clearly shows that the PCR of the plasmid did not work correctly as only a band seen just under 3.0 kb. A band at 0.7 kb is seen for the PR3 product.
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w31/we2.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 22 | Agarose gel analysis on samples after PCR plasmid linearization and overlap extension. </b> Samples after plasmid linearization (25_P3 (lin.)) and overlap extension (PR3 (p.e.)). A band is seen between 0.5 an 1kb. However we are unable to say that the overlap extension was successful, due to the small change in length when it is successful. No band is seen at 6800 bp (25_P3 lin.) implying that the plasmid linearization was not successful. Agarose gel (1%), ran for 45 hour at 120 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator. </figcaption>
                  <br>
                  </li>
                  <li> Redo of PCR reaction for plasmid linearization and overlap extension. This should result in the parts we need for Gibson assembly. We used a different polymerase (Q5) and increased the extension time from 3:40 to 5:00. </li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> PCR cleanup, followed by concentration determination via Nanodrop Spectrophotometer. As we saw from the DNA concentrations, clearly one of the 2 PCR reactions did not work.</li>
                  <li> To check if the PCR was correct, we put the PCR product on an Agarose gel. We expect clear bands at 0.7 kb (PR3) and 6.7 kb (plasmid) (Figure 29). Gel clearly shows that the PCR of the plasmid did not work correctly as only a band seen just under 3.0 kb. Moreover, we added all of the remaining primer extended gBlock from 01-08 on the gel as well to cut out the correct band and extract it from the gel. 
            <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w31/th.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 23 | Agarose gel analysis on samples after PCR plasmid linearization and overlap extension. </b> Samples after plasmid linearization (25_P3 (lin.)) and overlap extension (PR3 (p.e.)). Clear bands are seen between 0.5 an 1kb for PR3. However we are unable to say that the overlap extension was successful, due to the small change in length when it is successful.. No band is seen at 6800 bp (25_P3 lin.) implying that the plasmid linearization was not successful. Agarose gel (1%), ran for 45 hour at 120 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                  <li> Gel extraction (Qiagen kit) was used to purify the primer extended DNA (0.7kb, PR3) from the gel. Subsequently, the concentrations were determined using the Nanodrop Spectrophotometer.</li>
                  <li> As the PCR reaction again did not work, we tried another time. Redo of PCR reaction for plasmid linearization. This should result in the part we need for Gibson assembly. We again used Q5 polymerase and the increased extension time of 5:00. Moreover, we used a primer annealing temperature that will gradually increase with 1॰ C each cycle until 72॰ C, starting from 57॰ C.</li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> PCR cleanup, followed by concentration determination via Nanodrop Spectrophotometer.</li>
                  <li> To check if the PCR reaction was successful, we put the PCR product on an Agarose gel. We expect clear band at 6.7 kb (plasmid) (Figure 24). Gel clearly shows that the PCR of the plasmid did not work correctly as only a band is seen just under 3.0 kb.
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w31/fr.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 24 | Agarose gel analysis on samples after plasmid linearization via PCR. </b> Samples of 25_P3 after plasmid linearization. No band is seen at 6800 bp (25_P3 lin.) implying that the plasmid linearization was not successful. Agarose gel (1%), ran for 45 hour at 120 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                  <li> Ordered new primers, which will be used for overlap extension to introduce a new restriction site. </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 33</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Design a new cloning strategy for acquiring 31_PR3.                                                     </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Designed a new cloning strategy, making use of overlap extension PCR to build in a new restriction site in the gBlock of PR3. Ordered new primer iGEM 020 (Figure 25) (<a class="link" href=" https://2022.igem.wiki/tu-eindhoven/primers_gblocks">Primer list</a>). 
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/engi-suc/15.png"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 25 | New cloning strategy (Ligation after overlap extension). </b> As the Gibson Assembly strategy did not work, we wanted to a new ligation strategy. We designed a new primer to introduce a new restriction site and came up with the strategy which is seen in the figure. </figcaption>
                  <br>
                  </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 34</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Attempt the new cloning strategy to acquire the 31_PR3 plasmid.                                                        </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">		
                    <b>Tuesday</b>
                <ul>
                  <li> Prepared overlap extension primer iGEM 020 to working condition stock (10 µM) from the main stock (100 µM)</li>
                  <li> Started the PCR reaction for overlap extension. This results in the extended DNA sequence including the wanted restriction site (Klf1)</li>
                  <li> Subsequently started PCR cleanup, followed by concentration determination via Nanodrop Spectrophotometer. Products were stored at 4॰ C in the fridge.</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Restriction digest on primer extended plasmid (31_P3) and insert (PR3). For digestion, Klf1 (TF) and EcoRI HF (NEB) are used. rSAP, a phosphatase, was only added to the plasmid for 5’ and 3’ dephosphorylation</li>
                  <li> DNA gel electrophoresis was used to check if digestion and PCR reaction were successful and separate the different restriction products (Figure 26). The 2-log NEB ladder was added as reference. As gel did not show clear bands as expected, redid the digestion later today. Still continued this experiment by gel extraction and ligation.
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w34/we.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 26 | Agarose gel analysis on samples after enzymatic digestion. </b> For digestion of 25_P3 and PR3, KlfI (ThermoFisher) and EcoRI-HF (NEB) were used. A clear band is seen at 6700 bp for linearized 25_P3. A very big smear is seen for the digested PR3 (expected 663 bp), resulting in us repeating the experiment. Agarose gel (1%), ran for 45 hour at 120 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                  <li> Gel extraction (Qiagen kit) was used to check if digestion was correct and purify the DNA from the gel. Subsequently, the concentrations were determined using the Nanodrop Spectrophotometer. </li>
                  <li> Ligation of digested products using T4 DNA Ligase (NEB). Ligated product was stored in fridge overnight (4 ॰C).</li>
                  <li> Redo of Overlap extension PCR and subsequently started PCR cleanup, followed by concentration determination via Nanodrop Spectrophotometer. Products were stored at 4॰ C in the fridge.</li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Restriction digest on primer extended plasmid (31_P3) and insert (PR3). For digestion, Klf1 (TF) and EcoRI HF (NEB) are used. rSAP, a phosphatase, was only added to the plasmid for 5’ and 3’ dephosphorylation</li>
                  <li> DNA gel electrophoresis was used to check if digestion was correct and separate the different restriction products. (Figure 27). The 2-log NEB ladder was added as reference. Gel again did not show the expected results. We did not continue with this cloning strategy. 
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w34/th.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 27 | Agarose gel analysis on samples after enzymatic digestion. </b> For digestion of 25_P3 and PR3, KlfI (ThermoFisher) and EcoRI-HF (NEB) were used. A clear band is seen at 6700 bp for linearized 25_P3. No band is seen for the digested PR3 (expected 663 bp). The lower part of the ladder is also missing, suggesting that the gel ran for too long. Agarose gel (1%), ran for 45 hour at 120 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Transformation of ligated product in TOP10 Competent Cells. Negative and postive results, consisting of no insert and SEAP plasmid, respectively, were added. Transformants were plated on (prewarmed) LB-Agar plates and grew over the weekend at room temperature.</li>
                  <li> As we were not very hopefull for the ligation, and wanted to add an HA-tag before PR3 in our receptor design, we ordered 3 new gBlocks (HA-tagged PR3, GFP and UTR-GFP) and multiple primers (iGEM 021 – iGEM 026) (<a class="link" href=" https://2022.igem.wiki/tu-eindhoven/primers_gblocks">Primer list</a>).</li>
                  <li> We therefore designed a new cloning strategy for acquiring 0_HA_PR3, 8_HA_PR3 and 31_HA_PR3. Figure 28 shows the strategy for acquiring 8_HA_PR3. Similar strategies are used for the other plasmids. 
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/engi-suc/19.png"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 28 | New cloning strategy (Ligation with new gBlock).</b> As the previous cloning strategy did not work, we wanted to a new ligation strategy. We designed a new gBlock to introduce a new HA-tag next to PR3. </figcaption>
                  <br>
                  </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 35</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Preparations for Challenge Day and start with the part improvement experiments.                                                        </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Checked transformation of 31_PR3. Plates did not show any colonies on the cloning and negative control samples. Positive controls showed colonies.</li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Kanamycin Stocks (1000x) were made.</li>
                  <li> LB-Agar plates containing Kanamycine were poured for glycrol stock revival.</li>
                  <li> LB-Agar plates containing kanamycine and IPTG were poured for the challenge day.</li>
                  <li> LB-Agar plates containing chloramphenicol were poured, which will be used for our part improvement.</li>
                  <li> Making extra LB-Growth medium for the challenge day. </li>
                  <li> Reviving the glycerol stocks of E. Coli containing plasmids for different colored fluorescent proteins. (eYFP, mCherry, GFP, mOrange, eCFP and Neptune)</li>
                  <li> Reviving the glycerol stock containing the SEAP plasmid, as we were running out of this plasmid. </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Started small cultures from the colonies on the LB-agar plates from the transformation for the challenge day of yesterday (Figure 29).
                  <br><br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w35/we.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 29 | LB-Agar plates.  </b> BL21 cells transformed with either pET-28a GFP, mCherry, Neptune, eYFP, or mOrange plasmid. pET-28a eCFP was not photographed.  </figcaption>
                  <br>
                  </li>
                  <li> Started small cultures from the colonies on the LB-agar plates from the transformation of the SEAP plasmid (Figure 30).
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w35/we2.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 30 | LB-Agar plates. </b> DH5α cells transformed with pLS13 (SEAP) plasmid.</figcaption>
                  <br>
                  </li>
                  <li> Dissolving the plasmid PsB1C3 from the distribution kit of 2022. </li>
                  <li> Transformation of PsB1C3 plasmid into DH5a cells. </li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Created 22 glycerol stocks for each color of E. coli strain (Figure 31).
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w35/th.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 31 | Challenge Day preparations. </b> Twenty-two glycerol stocks for six colors were prepared, which were needed for the E. coli art during the Challenge Day.</figcaption>
                  <br>
                  </li>
                  <li> Started small cultures from the colonies on the LB-agar plates from the transformation of the part improvement (Figure 32).
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w35/th2.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 32 | LB-Agar plates </b> DH5α cells transformed with pSB1C3 plasmid.</figcaption>
                  <br>
                  </li>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures of SEAP we started yesterday and subsequent concentration determination by Nanodrop spectrophotometer. </li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures of the part improvement cells (pSB1C3) we started yesterday and subsequent concentration determination by Nanodrop spectrophotometer. </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 36</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Continue with part improvement experiments and start new cloning for the receptor construct plasmids which include the HA-tag. Test the challenge day glycerol stocks for its color.  </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">		
                <b>Wednesday</b>
                <ul>
                  <li> Dissolving gBlocks (gBlock 05 – gBlock 07) and primers (iGEM 021 – 026) to final concentrations. Final concentration for the gBlock is 25 ng/µL. Final molarity of primers are 100 µM (main stock) and 10 µM (working concentration). </li>
                  <li> gBlock (gBlock 05 - 07) amplification via PCR, followed by PCR cleanup and subsequent concentration determination via Nanodrop Spectrophotometer.</li>
                  <li> Restriction digest on multiple plasmids and inserts are performed (Table 2). rSAP, a phosphatase, was only added to the plasmid for 5’ and 3’ dephosphorylation.
                  <br><br>
                  <figcaption style="font-size:13px"><b> Table 2 | Overview of the restriction digests </b> Table shows plasmids from which the inserts were obtained as well as which restriction enzymes were used in order to cut the insert out of the plasmid </figcaption>
                  <br>
                  <center><img class="image-tofull" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w36/t2.png"></center>
                  <br>
                  </li>
                  <li> DNA gel electrophoresis was used to check if digestion and PCR reaction were successful and separate the different restriction products. (Figure 38). The 2-log NEB ladder was added as reference. 
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w36/we1.jpg"></center>
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w36/we2.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 33 | Agarose gels for DNA separation of digested plasmids and inserts. </b> After digestion of HA_PR3 (BamHI and EcoRI, 696 bp), pSB1C3 (EcoRI and SpeI, 2050 bp), GFP and UTR+GFP (EcoRI and SpeI, 923 bp and 963 bp, respectively), 2_P3 and 4_P3 (BamHI and EcoRI, around 6650 bp), HA_PR3 (KlfI and EcoRI, 718 bp) and 25_P3 (KlfI and EcoRI, around 6650 bp). Clear, expected bands were seen for all samples, except for pSB1C3. Here a band is seen at 3000 bp, however pSB1C3 is around 2000 bp. The outlined DNA bands were cut out and purified. Agarose gel (1%), ran for 1 hour at 100 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator.</figcaption>
                  <br></li>
                  <li> Gel extraction (Qiagen kit) was used to purify the correct bands of each lane from the gel. Because of time pressure, we were not able to determine the concentrations before ligation. </li>
                  <li> Ligation of digested products of today using T4 DNA Ligase (NEB). Incubated reaction overnight at 4 ॰C. This ligates the parts 0_HA_PR3, 8_HA_PR3, 31_HA_PR3, GFP and UTR_GFP.</li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Transformation of ligated product in TOP10 Competent Cells. Transformants were plated on (prewarmed) LB-Agar plates and grew overnight at 37 ॰C. </li>
                  <li> Redo of ligation of yesterday as we expected that we made some errors. Ligation of digested products of today using T4 DNA Ligase (NEB). Incubated reaction overnight at 4 ॰C. This ligates the parts 0_HA_PR3, 8_HA_PR3, 31_HA_PR3, GFP and UTR_GFP.</li>
                  <li> Tested the glycerol stocks we created for the Challenge day by plating on Kan-IPTG plates. </li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Transformation of ligated product of GFP and UTR_GFP in TOP10 Competent Cells. Transformants were plated on (prewarmed) LB-chloramphenicol plates and grew over the weekend at room temperature. </li>
                  <li> Small cultures were made from the transformants of yesterday; 0_HA_PR3, 8_HA_PR3, 31_HA_PR3, GFP and UTR_GFP (Figure 34).
                  <br><br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w36/fr1.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 34 | LB-Agar plates. </b> TOP10 cells transformed with either 0_HA_PR3, 8_HA_PR3, 31_HA_PR3, GFP, or UTR_GFP plasmid. </figcaption>
                  <br>
                  </li>
                  <li> Another DNA gel electrophoresis was used to check the exact plasmid we used for the part improvement, as we saw strange bands on Wednesday. We also checked if our ligation products were correct (Figure 35). The 2-log NEB ladder was added as reference. As a very small amount of DNA of ligation products was added as there was a small amount of sample left, requiring us to drastically increase the contrast making the image a bit vague. As seen on the gel, we received a different plasmid from what we expected. Moreover, we see that the ligation still was successful.  
                  <br><br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w36/fr2.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 35 | Agarose gel analysis on samples after enzymatic digestion.  </b> As pSB1C3 gave unexpected results on the last gel, we did an analysis on the plasmid. pSB1C3 is ran on a gel in a non-digested, 1x digested and 2x digested form. Again, the linear (1x digestion) plasmid shows a clear band at 3000 bp, meaning that the plasmid we received is not pSB1C3. Agarose gel (1%), ran for 45 hour at 120 V in 1x TAE buffer, stained with SYBR Safe DNA gel stain. Gel analysis was performed using a 470-nm blue light illuminator. </figcaption>
                  <br>
                  </li>
                  <li> Readout of Challenge Day test plates (Figure 36).
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w36/fr3.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 36 | LB-Agar plates. </b> DH5α cells transformed with pET-28a GFP, mCherry, Neptune, eYFP, mOrange and eCFP plasmids.  </figcaption>
                  <br>
                  </li>
                </ul>
                
                <b>Saturday</b>
                <ul>
                  <li> Alexander (our supervisor) took out the small cultures started yesterday and put them in the fridge at 4 ॰C. </li>
                </ul>
                
                </div>
            </div>
            
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 37</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Finalize the cloning of receptor constructs’ plasmids including HA-tag. Produce enough plasmid stocks for all upcoming cell experiments. Challenge day E. coli art. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Started new small cultures (30 mL) from the small cultures which were put in the fridge last Friday.</li>
                  <li> Started small cultures from the colonies on the LB-agar plates from the transformation for the GFP and UTR_GFP of Friday.</li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Plasmid preparation (Qiagen kit) of the small cultures we started yesterday followed by concentration determination by Nanodrop spectrophotometer.</li>
                  <li> Made sequencing samples of all plasmids prepped today. These will be sent to BaseClear for sequencing.</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> E. coli art during the Challenge Day. These fine artworks can be viewed at the <a class="link" href="education">education wiki page</a>. </li>
                </ul>
              
                <b>Friday</b>
                <ul>
                  <li> Transformation of part improvement plasmids (GFP & UTR+GFP) in Subcloning Efficiency™ DH5α™ Chemically Competent Cells. Transformants were plated on (prewarmed) LB-Agar plates and grew over the weekend at room temperature. </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 38</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i>  Finalize experiments for Part Improvement.                                                              </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> LB growth medium was made. </li>
                  <li> 2x YT growth medium was made.</li>
                  <li> As only 1 transformation was successful, redid the transformation of last Friday. Transformation of part improvement plasmids (GFP & UTR+GFP) in BL21 (DE3) Chemically Competent Cells. Transformants were plated on (prewarmed) LB-Agar plates and grew overnight at 37 ॰C. </li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Made LB-Agar medium to pour plates. Made new LB-Agar plates containing either ampicillin or chloramphenicol. </li>
                  <li> As again only 1 transformation was successful, redid the transformation of Yesterday. Transformation of part improvement plasmids (GFP & UTR+GFP) in BL21 (DE3) Chemically Competent Cells. Transformants were plated on (prewarmed) LB-Agar plates and grew overnight at 37 ॰C.</li>
                  <li> Autoclaved two 500 mL Erlenmeyer’s to sterilize them for usage in Large Cultures. </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> As no cultures were seen at 09:00, we redid the transformation again. We kept the plates from yesterday in the incubator, and at 18:00 we saw that colonies grew (Figure 37). Transformation of part improvement plasmids (GFP & UTR+GFP) in BL21 (DE3) Chemically Competent Cells. Transformants were plated on (prewarmed) LB-Agar plates (chloramphenicol or ampicillin) and grew overnight at 37 ॰C.
                  <br><br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w38/we1.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 37 | LB-Agar plates. </b> BL21 cells transformed with either GFP, or UTR+GFP plasmid. </figcaption>
                  <br>
                  </li>
                  <li> Started small cultures from the colonies on the LB-agar plates from the transformation of last Friday (GFP and UTR-GFP) and grew overnight at 37 ॰C, 250 rpm (Figure 37). </li>
                  <li> </li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Transformants of yesterday only grew on ampicillin plates.
                  <br><br>
                  <center><img class="w-100" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w38/th1.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 38 | LB-Agar plates. </b> BL21 cells transformed with either pSB1C3, GFP, or UTR+GFP plasmid. </figcaption>
                  <br>
                  </li>
                  <li> Started large cultures from small cultures (GFP and UTR+GFP) started yesterday. Samples from the culture medium were taken every 20 minutes after induction via IPTG. After 6 hours, fluorescence was measured of each sample (Figure 39).  
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/bl/w38/th2.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 39 | Visual comparison xpression GFP and UTR-GFP </b>Samples of the GFP (right) and UTR+GFP (left) large cultures put under 470-nm blue light illuminator.</figcaption>
                  <br>
                  </li>
                  <li> Remade small cultures from the plates of yesterday and grew overnight at 37 ॰C, 250 rpm. </li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Made glycerol stocks of the small cultures started yesterday. </li>
                </ul>
                </div>
            </div>
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 39</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Repeat the Part improvement experiments.                                                               </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Revived the glycerol stocks made on last Friday (GFP, UTR+GFP).</li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Started small cultures from the plates we made yesterday.</li>
                </ul>
            
                <b>Wednesday</b>
                <ul>
                  <li> Started large cultures from small cultures (GFP and UTR+GFP) started yesterday. Samples from the culture medium were taken every 20 minutes after induction via IPTG. After 6 hours, fluorescence was measured of each sample  </li>
                </ul>
                </div>
            </div>
            
                            </div>
                          </div>
                      </div>
                    </div>
            
                    <h1>CellLab</h1><br>
            
                  <div class="row gy-5 gx-0 pb-5">
                    <div class="col-1 d-flex justify-content-center">
                        <div class="bar" style="height: 100%;"></div> 
                    </div>
                      <div class="col-10">
                        <div class="row gx-5">
                          <div class="col-12 mb-3">
                            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 25</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Start our own HEK293T cell lines from the cells stored in the -80 ॰C. Pass our HEK293T cell lines.  </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Made complete MDEM medium.</li>
                  <li> Thawed HEK293T cells (P. 20) from an aliquot in the -80 ॰C. </li>
                  <li> </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 21).</li>
                  <li> Froze an aliquot of these HEK293T cells.</li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 22).</li>
                </ul>
                </div>
            </div>
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 26</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> First SEAP assay for the Receptor transfection and activation analysis. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 23). </li>
                  <li> Start with the SEAP cell experiment by seeding cells on a plate.</li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Transfection of plasmid DNA (pLeo619, pLS13, pLS15) to the seeded cells, followed by the addition of ligand molecules</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 24).</li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Executed the SEAP assay on the cell medium. See results page for the results.</li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 25).</li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 27</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> First IL-10 ELISA to verify therapeutic expression of IL-10. Redo of last weeks’ SEAP assay to verify receptor transfection, and acitvition.  </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 26). </li>
                  <li> Start with the cell experiments (SEAP or IL-10 expression by RR-120 addition) by seeding cells on a 24-well plate.</li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Transfection of plasmid DNA (pLeo619, pLS15 and either pLS13 or pLS13-IL-10) to the seeded cells, followed by the addition of ligand molecules.</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 27). </li>
                  <li> Start of IL-10 ELISA assay. Seeding the plates with antibodies.</li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 28). </li>
                  <li> Executed the rest of the ELISA assay. See results page for the results</li>
                  <li> Snapfreezed the SEAP assay samples, taken on the cell medium. </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 28</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Perform another IL-10 ELISA, in which IL-10 expression was quantified.  </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 29). </li>
                  <li> Start with the cell experiments (IL-10 expression RR-120 addition) by seeding cells on a 24-well plate.</li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Transfection of plasmid DNA (pLeo619, pLS15 and pLS13-IL-10) to the seeded cells, followed by the addition of ligand molecules.</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 30). </li>
                  <li> Start of IL-10 ELISA assay. Seeding the plates with antibodies.</li>
                  <li> Executed the SEAP assay using the snap-freezed samples of last week's Thursday.</li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Executed the rest of the ELISA assay. See results page for the results.</li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 31). </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 29</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Pass our HEK293T cultures as no cell experiments were planned. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 32). </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 33). </li>
                </ul>
            
                <b>Friday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 34). </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 30</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Start with next part of our research; Receptor activation via antibodies.  </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 35). Continued with this culture to train the other team members.</li>
                  <li> Received a newer HEK293T cells for the cell experiments. Passage of the HEK293T cells (P. 12). </li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 13). </li>
                  <li> Start with the cell experiment (SEAP expression by anti-PR3 addition) by seeding cells on a 24-well plate.</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Transfection of plasmid DNA (either 0_PR3 or 8_PR3, pLS13 and pLS15) to the seeded cells, followed by the addition of ligand molecules.</li>
                  <li> Passage of the HEK293T cells (P. 36). Continued with this culture to train the other team members.</li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 14). </li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Executed the SEAP assay on the cell medium. See results page for the results</li>
                  <li> Passage of the HEK293T cells (P. 37). Continued with this culture to train the other team members.</li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 31</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Repeat last weeks’ validation of antibody-induced receptor activation. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 38). Continued with this culture to train the other team members.</li>
                  <li> Passage of the newer HEK293T cells (P. 15). </li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Start with the cell experiment (SEAP expression by anti-PR3 addition) by seeding cells on a 24-well plate.</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Transfection of plasmid DNA (either 0_PR3 or 8_PR3, pLS13 and pLS15) to the seeded cells, followed by the addition of ligand molecules.</li>
                  <li> Passage of the newer HEK293T cells (P. 16). </li>
                  <li> Passage of the HEK293T cells (P. 39). Continued with this culture to train the other team members.</li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Executed the SEAP assay on the cell medium. See results page for the results.</li>
                  <li> Passage of the HEK293T cells (P. 17). </li>
                  <li> Passage of the HEK293T cells (P. 40). Continued with this culture to train the other team members.</li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 32</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Pass our HEK293T cell cultures as no cell experiments were planned (including pictures!). </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 18). Figure 1 shows HEK293T cells before the passage protocol. Cells show to be sticking to the plate with a confluence of 70 %. Figure 2 shows the cells after the addition of Trypsin, which will detatch the cells from the culture flask. Small clumps of cells are still seen. Figure 3 shows the cells after passage. Cells show to be round, as they are not yet attached to the culture flask. 
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/cl/w32/mo1.png"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 1 | Attached HEK293T cells with 70% confluency. </b>Image captured with 4x optical zoom.</figcaption>
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/cl/w32/mo2.png"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 2 | Detached HEK293T cells after the addition of Trypsin.  </b>Image captured with 4x optical zoom.</figcaption>
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/cl/w32/mo3.png"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 3 | Detached HEK 293T cells after transferring to a new culture flask and medium.  </b>Image captured with 4x optical zoom.</figcaption>
                  <br>
                  </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 19). </li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 20). </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 34</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Flow cytometry experiment to check if the antibodies have affinity for the PR3 affinity domain on the receptor. Verify if IL-10 expression has an effect on the intracellular pathway. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Passage of HEK293T cells (P. 24)</li>
                  <li> Start with the Flow Cytometry experiment (binding of anti-PR3 to PR3 on the receptor) by seeding cells on a 24-well plate.</li>
                  <li> </li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Transfection of plasmid DNA (0_PR3, 8_PR3 and eGFP) to the seeded cells. No ligands are added at this stage.</li>
                  <li> Start with the SEAP experiment to check if the addition of IL-10 has an effect on the function of STAT 3 by seeding cells on a 24-well plate.</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Passage of HEK293T cells (P. 25)</li>
                  <li> Transfection of plasmid DNA (pLeo619, pLS13 and pLS15) to the seeded cells, followed by the addition of ligand molecules.</li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Executed the Flow cytometry experiment to check if the anti-PR3 antibodies actually bind to the PR3 protein on our receptor. See results page for the results</li>
                </ul>
                
                <b>Friday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 26)</li>
                  <li> Executed the SEAP assay on the cell medium to check if the addition of IL-10 has an effect on the function of STAT 3.</li>
                </ul>
                </div>
            </div>
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 35</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Pass our HEK293T cell cultures as no cell experiments were planned. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 27)</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 28)</li>
                </ul>
            
                <b>Friday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 29)</li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 36</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Pass our of HEK293T cell cultures as no cell experiments were planned. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 30)</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> •	Passage of the HEK293T cells (P. 31)</li>
                </ul>
              
                <b>Friday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 32) Passed in three T75 flasks, as we were preparing for upcoming cell experiments next week. </li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 37</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Verify if antibodies bind to the different affinity domains on the receptor constructs using a Flow Cytometry experiment. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> All our cultured HEK293T cells died due to a bacterial infection which was present in the Cell Lab. Continued with a new culture of HEK293T cells kindly provided by one of our supervisors. </li>
                  <li> Start with the Flow Cytometry experiment (binding of anti-HA antibody and anti-PR3 on the receptor) by seeding cells on a 24-well plate.</li>
                  <li> Start with the cell experiment (SEAP expression by anti-HA addition) by seeding cells on a 24-well plate</li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> The cells on the 24-well plate of yesterday had died because of a shortage of DMEM growth medium. The cell experiment therefore was postponed to next week. </li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 14)</li>
                </ul>
            
                <b>Friday</b>
                <ul>
                  <li> Passage of the HEK293T cells (P. 15). Passed in two T75 flasks, as we were preparing for upcoming cell experiments next week.</li>
                </ul>
                </div>
            </div>
            
            
            
            
            
            
            <button class="accordionblue">
                <h5 class="week_notebook"><u>WEEK 38</u></h5>
                <p class="padding_notebook"><i>Weekly Goal: </i> Repeat last week experiment. Verify if antibodies bind to the different affinity domains on the receptor constructs using a Flow Cytometry experiment. </p>
                <div class="arrow-circle"></div>
            </button>
            <div class="panelblue">
                <div style="padding:1.25em 0.5em;">
                <b>Monday</b>
                <ul>
                  <li> As the bacterial infection was still not completely gone in the cell lab, our HEK293T cells died again. Continued with a new culture of HEK293T cells kindly provided by one of our supervisors. As we less cells than expected, we had to remove some of the conditions we planned on testing.</li>
                  <li> Start with the Flow Cytometry experiment (binding of anti-HA antibody and anti-PR3 on the receptor) by seeding cells on a 24-well plate.</li>
                  <li> Start with the cell experiment (SEAP expression by anti-HA addition) by seeding cells on a 24-well plate.</li>
                </ul>
                
                    <b>Tuesday</b>
                <ul>
                  <li> Transfection of plasmid DNA (either 0_HA_PR3 or 8_HA_PR3, pLS13 and pLS15) to the seeded HEK293T cells, followed by the addition of ligand molecules.</li>
                  <li> Tried an mRNA transfection in HEK 293 T cells (the mRNA was kindly provided by our sponsor RiboPro).</li>
                </ul>
                
                <b>Wednesday</b>
                <ul>
                  <li> Used fluorescence and brightfield microscope to check if our transfection of plasmids in HEK293T cells was successful (Figure 4).
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/cl/w38/we1.jpg"></center>
                  <br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/cl/w38/we2.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b> Figure 4 | Seeded HEK293T cells after transfection. </b> Fluorescence and brightfield microscope images. </figcaption>
                  <br>
                  </li>
                </ul>
                
                <b>Thursday</b>
                <ul>
                  <li> Due to the bacterial infection, some of our cells have died again. This is seen by the pinkish colour of the medium instead of yellow (indication that medium is not used by the cells)(Figure 5).
                  <br><br>
                  <center><img class="image-tofull" style="border-radius:10px" src="https://static.igem.wiki/teams/4160/wiki/lab/notebook/cl/w38/th1.jpg"></center>
                  <br>
                  <figcaption style="font-size:13px"><b>Figure 5 | Seeded HEK293T cells after transfection. </b> Pinkish wells indicate that HEK293T cells have died.  </figcaption>
                  <br>
                  </li>
                  <li> Executed the Flow cytometry experiment to check if the new polyclonal anti-PR3 antibodies (PR3 Polyclonal Antibody, PA5-115791, ThermoFisher) and Anti-HA antibodies (Alexa Fluor® 488 anti-HA.11 Epitope Tag Antibody, 901509, Biolegend) bind to the PR3 protein and HA on our receptor, respectively. Secondary antibody used for staining is Anti-Rabbit IgG (H+L) (ThermoFisher, Alexa Fluor™ 488, A-21206). See results wiki page for the results.</li>
                </ul>
                </div>
            </div>
                          </div>
                        </div>
                    </div>
                  </div>
            
                  </div>
              <div></div>
              </div>
            </div>

            <!--accordion-->
            <script>
              var acc = document.getElementsByClassName("accordionblue");
              var i;

              for (i = 0; i < acc.length; i++) {
                acc[i].addEventListener("click", function () {
                  /* Toggle between adding and removing the "active" class,
                  to highlight the button that controls the panel */
                  this.classList.toggle("active");

                  /* Toggle between hiding and showing the active panel */
                  var panel = this.nextElementSibling;
                  if (panel.style.maxHeight) {
                    panel.style.maxHeight = null;


                  } else {
                    panel.style.maxHeight = '100%';
                  }

                });
              }
</script>

            <script>
              const wikiToolsLink = document.getElementById('wiki-tools-link')
              const wikiToolsRepo = document.getElementById('wiki-tools-repo')
              const wikiToolsPipeline = document.getElementById('wiki-tools-pipeline')
          
              const path = document.location.pathname.split('/')
          
              const competitionSlug = document.location.host.split('.')[0]
              const teamSlug = path[1]
              const page = path[2] === '' ? 'index' : path[2]
          
              wikiToolsLink.href = `https://tools.igem.org/wiki-tools/${competitionSlug}/${teamSlug}/${page}`
              wikiToolsRepo.href = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}`
              wikiToolsPipeline.src = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}/badges/main/pipeline.svg`
          </script>

            <script src="static/bootstrap.bundle.min.js"></script>
            <footer id="svg-end" class="pt-5 pb-5 footer py-5 bg-dark text-white">
              <div class="container">
                <div class="row mb-4">
                  <div class="col-lg-6 col-xs-12">
                    <h4 class="mb-3" style="padding-left: 50px;"><a href="https://2023.igem.wiki/iiser-tvm/">Oasys</a></h4>
                    <p style="color: aliceblue;">A depression diagnostics kit envisioned by iGEM IISER TVM 2023</p>
                  </div>
                  <div class="col-lg-3 col-xs-12">
                    <h4 class="mt-lg-0 mt-sm-3">Links</h4>
                      <ul class="m-2 p-2">
                        <li><a href="https://biology.iisertvm.ac.in/">School of Biology, IISER TVM</a></li>
                        <li><a href="https://www.instagram.com/igem_iisertvm">Instagram</a></li>
                        <li><a href="https://twitter.com/igem_iisertvm">Twitter</a></li>
                        <li><a href="https://www.youtube.com/c/iGEMIISERTVM">YouTube</a></li>
                        <li><a href="https://2021.igem.org/Team:IISER_TVM">iGEM IISER TVM 2021</a></li>
                        <li><a href="https://2023.igem.wiki/iiser-tvm/">iGEM IISER TVM 2022</a></li>
                      </ul>
                  </div>
                  <div class="col-lg-3 col-xs-12">
                    <h4 class="mt-lg-0 mt-sm-4 mb-3 center">Contact</h4>
                    <p style="color: aliceblue";>IISER Thiruvananthapuram, Maruthamala PO, Vithura, Kerala, India - 695551</p>
                    <p class="mb-0"style="color: aliceblue">+91 0471 277 8009</p>
                    <p style="color: aliceblue">igem@iisertvm.ac.in</p>
                  </div>
                </div>
                <hr>
                <!-- The following MUST be on every page: license information and link to the repository on gitlab.igem.org -->
                <div class="row mt-4">
                  <div class="col">
                    <p class="mb-0"><small>© 2022 - Content on this site is licensed under a <a class="subfoot" href="https://creativecommons.org/licenses/by/4.0/" rel="license">Creative Commons Attribution 4.0 International license</a>.</small></p>
                    <p><small>The repository used to create this website is available at <a href="https://gitlab.igem.org/2022/iiser-tvm">gitlab.igem.org/2022/iiser-tvm</a>.</small></p>
                  </div>
                </div>
              </div>
            </footer>
           
            </body>
            </html>

            